sdk/cmake/fp-app.cmake

```diff
@@ -0,0 +1,112 @@
+if (NOT DEFINED TARGET)
+  set(TARGET "emulator")
+endif()
+
+function(fp_add_script_output target)
```

Comment:
Can you provide a description of this function?

Reply:
I think it should be a bit better now :)